File: C Functions Task/todo2/app1.c
#include <stdio.h>
//called function
void formula(int a,int b){
    int result=(a*a)+(b*b)+2*(a+b);
    print("%d",result);
}

int main()
{
    int a=10;
    int b=20;
    //function caller
    formula(a,b);
    return 0;
}

File: C Functions Task/todo3/app1.java
public class Main
{
	public static void main(String[] args) {
	   	System.out.print("Result:");
		//function caller
	   	formula(10,20);
	}
	//called function
	public static void formula(int a,int b){
	    int result=(a*a)+(b*b)+2*(a+b);
	    System.out.println(result);
	}
}
